Cython/Includes/cpython/object.pxd

```diff
@@ -231,7 +231,7 @@ cdef extern from "Python.h":
     # used to determine the result of this function.
 
     # Subclass determination is done in a fairly straightforward way,
-    # but includes a wrinkle that implementors of extensions to the
+    # but includes a wrinkle that implementers of extensions to the
```

Comment:
Both versions (implementer/implementor) are [valid](https://www.thefreedictionary.com/implementor).

Reply:
indeed... and was moved into "rare" dictionary in codespell just 3 days ago: https://github.com/codespell-project/codespell/commit/d46b6872ed7be34496b9cc52f444eb6cefdb7c20 . I will just add explicit skip for it for now... will refresh PR tomorrow

Reply:
done